TACC/Frontera-Portal
# Task/FP-652 - Add welcome messages to Manage Account and Unauthenticated User Ticket Page

- Create welcome messages for user's account management area "ManageAccount.js"
- Create welcome message for the unauthenticated users ticket submission page "TicketStandaloneCreate.js"

## Overview: ##
Added the grey Alert bars to the described pages. There seems to be an issue with the error returned when the Authenticated User dispatch fails and prevents the alert tab from being closed. This may be a bug that needs looking into.

## PR Status: ##


## Related Jira tickets: ##

* [FP-652](https://jira.tacc.utexas.edu/browse/FP-652)

## Summary of Changes: ##

## Testing Steps: ##
1. Log in
2. Go to user profile page `https://cep.dev/accounts/profile/`
3. check that the welcome alert shows up (and is dismissable)
4. Log out
5. Go to ticket creation page `https://cep.dev/tickets/new/`
6. check that the welcome alert show up (and is dismissable)

## UI Photos:
![unauthed_tickets](https://user-images.githubusercontent.com/29575979/97374813-38f27f00-1887-11eb-8eab-fb6ee97a18be.png)
![user_account](https://user-images.githubusercontent.com/29575979/97374815-3a23ac00-1887-11eb-8029-2fd6e8e2fc07.png)


## Notes: ##
